config.go

```diff
@@ -603,7 +604,9 @@ func LoadConfig(interceptor signal.Interceptor) (*Config, error) {
 	// Next, load any additional configuration options from the file.
 	var configFileError error
 	cfg := preCfg
-	if err := flags.IniParse(configFilePath, &cfg); err != nil {
+	fileParser := flags.NewParser(&cfg, flags.Default)
+	err := flags.NewIniParser(fileParser).ParseFile(configFilePath)
```

Comment:
Is this qualitatively different than the existing `IniParse` or just a long form way of executing the same logic?

Reply:
It is executing the same logic as IniParse

Reply:
It is executing the same logic as `IniParse` and we get a handle to the parse object.
```
func IniParse(filename string, data interface{}) error {
	p := NewParser(data, Default)

	return NewIniParser(p).ParseFile(filename)
}
```